### ISSUE TITLE
[bug] extension icon being duplicated or invisible in the browser

### ISSUE THREAD

**Issue by fallsdevil** [rijusougata13/wa-web-status-downloader#2]
I'm testing this extension in the Vivaldi browser on Windows 11, and I noticed the icon has a strange bug in which every time I deactivate/activate the extension itself or others, it is multiplied or becomes invisible, but changing the space in the same way as multiplies.

[Here](https://clipchamp.com/watch/A9yL7oqBldu) is a video I recorded of my laptop screen. (Sorry about the quality, I recorded it with the new Windows capture tool and it's not 100% functional).

**Comment by rijusougata13** [comment]
Hi, thanks for reporting the bug. Just wanted to confirm, if you delete this extension completely is the spacing issues still persist? it will help to debug if its been an issue for this extension or with the browser.

thank you

**Comment by fallsdevil** [comment]
Yes, I removed the extension, but it remained the same and the first time it had several duplicate icons, after removing and installing it again it became "invisible"
